2697. Lexicographically Smallest Palindrome/solution.py

```diff
@@ -0,0 +1,28 @@
+class Solution:
+    # Given a string w/ only lowercase English letters, turn the string into a 
+    # palindrome by making the smallest number of edits to the string and
+    # creating the lexographically smallest pallindrome. Lexographically smallest
+    # just means alphabetically smaller when each letter is compared from left to
+    # right.Return the pallindrome.
```

Comment:
I think you should proofread your comments a bit more O.o I noticed that both you and Cody spell "lexicographically" incorrectly; you miss the ic after the x.